jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/ExecuteCommandWarning.kt

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.ecs.exec
+
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.ui.Messages
+import com.intellij.ui.components.JBCheckBox
+import com.intellij.ui.components.JBLabel
+import com.intellij.ui.layout.panel
+import software.aws.toolkits.jetbrains.settings.EcsExecCommandSettings
+import software.aws.toolkits.jetbrains.utils.ui.visible
+import software.aws.toolkits.resources.message
+import javax.swing.JComponent
+
+class ExecuteCommandWarning(project: Project, enable: Boolean) : DialogWrapper(project) {
```

Comment:
This can be replaced with `com.intellij.openapi.ui.Messages#showYesNoDialog(com.intellij.openapi.project.Project, java.lang.String, java.lang.String, java.lang.String, java.lang.String, javax.swing.Icon, com.intellij.openapi.ui.DialogWrapper.DoNotAskOption)`

Reply:
Done